doc/energyplus_packages_windows.lst

```diff
@@ -0,0 +1,49 @@
+xetex
+amsfonts
+amsmath
+mathspec
+etoolbox
+iftex
+fontspec
+l3packages
+l3backend
+xkeyval
+geometry
+graphics
+graphics-cfg
+graphics-def
+grffile
+xcolor
+colortbl
+latex-tools
+framed
+booktabs
+multirow
+listings
+enumitem
+url
+hyperref
+ltxcmds
+pdftexcmds
+infwarerr
+kvsetkeys
+kvdefinekeys
+pdfescape
+hycolor
+letltxmacro
+auxhook
+kvoptions
+intcalc
+etexcmds
+bitset
+bigintcalc
+stringenc
+rerunfilecheck
+uniquecounter
+siunitx
+translations
+refcount
+gettitlestring
+miktex-dvipdfmx
+dvipdfmx
+zapfding

```

Comment:
A list of packages that E+ uses that aren't part of the essential package. compiled this list by looking at the output of a fresh essential install with AutoInstall set to 1 (by parsing the output log)